PyInstaller/hooks/hook-win32ctypes.core.py

```diff
@@ -0,0 +1,26 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2020, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
+
+# TODO: remove this hook during PyInstaller 4.5 release cycle!
+
+from PyInstaller.utils.hooks import exec_statement, collect_submodules
+
+# We need to collect submodules from win32ctypes.core.cffi or
+# win32ctypes.core.ctypes for win32ctypes.core to work. The use of
+# the backend is determined by availability of cffi.
+cffi_available = exec_statement(
+    """try:import cffi;print('\\nTrue')\nexcept: print('\\nFalse')"""
+).split()[-1] == 'True'
```

Comment:
rc = exec_statement_rc("import cffi")
cffi_available = (rc == 0)

Reply:
This seems like a good idea. We currently don't seem to have `exec_statement_rc` - I'll open a separate pull request to add it, and then revisit this.

Reply:
Hmm, actually, now that I've tried this with `exec_statement_rc` implementation using `exec_python_rc`... 
attempting to import a non-existing module prints an error message to stderr, which is undesirable in this case. And trying to work around this using try-except block and `sys.exit()` will make the code very similar to the existing one (sans the post-processing part).

Reply:
Now that you even implemented `can_import_module`, you most probably want to use that? :-)

Reply:
Yeah, that's the plan. :-)